x-pack/plugin/core/src/main/resources/logs-default-policy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "phases": {
+    "hot": {
+      "min_age": "0ms",
+      "actions": {
+        "rollover": {
+          "max_size": "50gb",
+          "max_age": "30d"
+        }
+      }
+    }
+  }
```

Comment:
Is there a way to add metadata to the ILM policy so we also know which stack version initially loaded it?

Reply:
Not currently, we could potentially add it in the future though